dinky-gateway/src/main/java/org/dinky/gateway/kubernetes/operator/KubetnetsApplicationOperatorGateway.java

```diff
@@ -90,12 +98,17 @@ public GatewayResult submitJar(FlinkUdfPathContextHolder udfPathContextHolder) {
                                             .getStatus()
                                             .getJobStatus()
                                             .getJobId();
+                                    String jobName = flinkDeployment1
+                                            .getStatus()
+                                            .getJobStatus()
+                                            .getJobName();
+                                    if(Asserts.isNull(jobName)){
+                                        logger.warn("waiting for job init");
+                                        return false;
+                                    }
                                     result.setJids(Collections.singletonList(jobId));
                                     return true;
                                 }
-                                if (status.equals("DEPLOYING")) {
-                                    getK8sClientHelper().createDinkyResource();
```

Comment:
`getK8sClientHelper().createDinkyResource();` can't be deleted, it must be executed after the task has been submitted to k8s and after the deploymen can be retrieved